torch/_weights_only_unpickler.py

```diff
@@ -144,6 +144,8 @@ def _tensor_rebuild_functions():
         torch._utils._rebuild_meta_tensor_no_storage,
         torch._utils._rebuild_nested_tensor,
         torch._utils._rebuild_wrapper_subclass,
+        # Allowlisting this, but not allowlisting the numpy functions by default
+        torch._utils._rebuild_device_tensor_from_numpy,
```

Comment:
Is this because this function is always safe and help when a user is trying to allow numpy-based serialization?
I think we could be clearer on the intent here

Reply:
I think we should allowlist functions provided in `torch._utils` for rebuild, but as we discussed I don't think we want to allowlist the numpy functions themselves